deploy/clowdapp.yaml

```diff
@@ -514,6 +518,9 @@ parameters:
 - name: NEWER_RELEASEVER_CSAF
   description: Enable finding updates/cves in newer releaseversion from CSAF
   value: "true"
+- name: VMAAS_VERSION_FILE_PATH
+  description: Path to the VMaaS VERSION file
+  value: "../VERSION"
```

Comment:
I don't think this is a correct path. File is probably not even present in the container image, it needs to be copied in Dockerfile